firefox-ios/Client/Frontend/Home/Pocket/PocketStandardCell.swift

```diff
@@ -51,8 +47,7 @@ class PocketStandardCell: UICollectionViewCell, ReusableCell {
 
     private lazy var sponsoredLabel: UILabel = .build { label in
         label.adjustsFontForContentSizeCategory = true
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .caption2,
```

Comment:
@cwzilla The sponsored label was previously size 12, but using the new system caption2 is now size 11. Can you confirm if we're okay with the update?

Reply:
Could we actually increase it to `caption1`?